v2_changes/applications.adoc

```diff
@@ -9,15 +9,16 @@
 
 toc::[]
 
-== Overview
-Applications have always been a focal point within OpenShift.  In OpenShift v2, an application was very well defined in that it consisted of 1 web framework and no more than 1 of any given cartridge type.  So you could have 1 php and 1 mysql for example.  But you couldn't have 1 ruby, 1 php, and 2 mysqls.  You also couldn't have a mysql by itself.
+== Applications in OpenShift v2
+Applications have always been a focal point within OpenShift. In OpenShift v2, an application was very well defined in that it consisted of one web framework and no more than one of any given cartridge type. So an application could have one PHP and one MySQL, for example, but it could not have one Ruby, one PHP, and two MySQLs. It also could not have a MySQL cartridge by itself.
 
-The limited scoping for applications meant that OpenShift could perform seamless linking for all components within an application via well defined environment variables.  Every web framework knew how to connect to mysql with OPENSHIFT_MYSQL_DB_HOST and OPENSHIFT_MYSQL_DB_PORT for example.  But this linking was limited to within an application and only worked within cartridges designed to work together.  There wasn't anything to help linking across application components such as sharing a mysql across 2 applications.
+The limited scoping for applications meant that OpenShift could perform seamless linking for all components within an application using well-defined environment variables. Every web framework knew how to connect to MySQL using the `OPENSHIFT_MYSQL_DB_HOST` and `OPENSHIFT_MYSQL_DB_PORT` variables, for example. But this linking was limited to within an application and only worked within cartridges designed to work together. There was nothing to help link across application components, such as sharing a MySQL instance across two applications.
 
-What was clear from v2 is that solving the problems of the entire application is essential.  Most other PaaSes limit themselves to web frameworks and rely on external services for the other types of components.  OpenShift v3 takes the next steps making even more application topologies possible and existing topologies more manageable.
+== Rethinking "Applications" for OpenShift v3
```

Comment:
I really don't like this title, and don't like the quotes in titles, but we'll rename and restructure later.